soundchip.js

```diff
@@ -67,20 +67,14 @@ define(['./utils'], function (utils) {
         function toneChannel(channel, out, offset, length) {
             var i;
             var reg = register[channel], vol = volume[channel];
-            if (reg === 1) {
```

Comment:
Glad to see this go: though I don't remember the reason I thought this was required. I'm sure it fixed an audio glitch way back but I can't find any reference to it in git history: so... this is an improvement :)